README.md

```diff
@@ -7,9 +7,9 @@ This operator is the entrypoint for installing the Migration Toolkit for Red Hat
 - mtRHO is compatible with OpenShift 4.10.7/4.10.7+ versions.
 - Go 1.18 version needed to build operator
 
-## Prerequisites
+## Dependencies
 
-1. Install pipeline operator. [Here](https://docs.openshift.com/container-platform/4.10/cicd/pipelines/installing-pipelines.html) are the steps to install pipeline operator.
+1. mtRHO installs pipeline operator as dependency in `<installation>` namespace. (**Note:** If this operator is already installed in some other namespace than it won't be again installed by installing mtRHO. Uninstalling mtRHO will not uninstall pipeline operator as well.)
```

Comment:
```suggestion
1. mtRHO installs pipeline operator as dependency in `<installation>` namespace. (**Note:** This will not impact any existing installation of the pipeline operator. Uninstalling mtRHO later also will not impact the pipeline operator.)
```